### PR TITLE
CLC-5609 Catch up with changed links at the ETS web site

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/CLC-5609

### CODE REVIEW

File: src/assets/templates/academics.html [hunk 1]
```diff
@@ -39,7 +39,7 @@ <h1 class="cc-heading-page-title">My Academics</h1>
       This page displays academic status information for UC Berkeley students and faculty, and is not available to staff.
     </p>
     <p>
-      If you believe you are seeing this message is an error, please contact <a href="https://ets.berkeley.edu/help/calcentral">CalCentral Help</a>.
+      If you believe you are seeing this message is an error, please <a href="http://www.ets.berkeley.edu/discover-services/calcentral/students-getting-started">let us know</a>.
```
Comment (johncrossman) [inline src/assets/templates/academics.html]: @raydavis did not change this wording but, IMO, it needs changing: "If you believe you are seeing this message is an error, please..."

Comment (raydavis) [inline src/assets/templates/academics.html]: MO agrees with YO, @johncrossman . I'll amend.

Comment (johncrossman) [inline src/assets/templates/academics.html]: And less urgent: "http:" should be "https:"

Comment (raydavis) [inline src/assets/templates/academics.html]: That is purposeful, @johncrossman, although maybe I'm mispurposed. If URL paths at the website have to be redirected to another host (for example, to point to the Knowledge Base), an "https:" protocol will keep it from going through. A dozen times burned, once shy.

File: src/assets/templates/footer.html [hunk 2]
```diff
@@ -22,9 +22,8 @@
         </a>
       </span>
       <a href="//security.berkeley.edu/policy">Usage Policy</a>
-      <a href="https://ets.berkeley.edu/category/category-tags/calcentral">About</a>
-      <a href="https://ets.berkeley.edu/forms/request-calcentral-support-or-give-feedback">Feedback</a>
-      <a href="https://ets.berkeley.edu/calcentral">Help</a>
+      <a href="http://www.ets.berkeley.edu/discover-services/calcentral">About</a>
+      <a href="http://www.ets.berkeley.edu/discover-services/calcentral/students-getting-started">Feedback</a>
```
Comment (johncrossman) [inline src/assets/templates/footer.html]: No more "Help" for the masses?

Comment (raydavis) [inline src/assets/templates/footer.html]: The "Help" link pointed to the exact same page as "About", and "About" reflects the page contents more accurately.

**Comment by johncrossman** [comment]
Thanks @raydavis – looks good. The code and copy that needs a tweak lives at www.ets.berkeley.edu and Service Now.

**Pull request merged**